### ISSUE TITLE
Prettify the test format

### ISSUE THREAD

**Issue by ChihChengLiang** [status-im/eth2-testgen#1]
The YAML file is pretty long now and it's hard to read and review. 

I've tried to minimize the length of validators' input by only reporting validators' status in YAML, as the following example shows.
```
fork: tchaikovsky
summary: Test vectors for shuffling a list based upon a seed using `shuffle`
test_suite: shuffle
title: Shuffling Algorithm Tests
version: 1.0
test_cases:
- input:
    crosslinking_start_shard: 210
    validators: [2, 4, 0, 0, 2, 2, 4, 2, 3, 1, 0, 3,
      3, 4, 4, 4, 1, 1, 1, 1, 3, 2, 3, 0, 2, 4, 0, 2,
      4, 0, 0, 4, 2, 1, 4, 1, 4, 2, 2, 1, 2, 4, 0, 4,
      0, 3, 0, 4, 4, 0, 0, 1, 3, 3, 0, 4, 3, 1, 1, 3,
      1, 0, 0, 1, 0, 0, 4, 1, 2, 0, 1, 4, 2, 1, 1, 4,
      1, 1, 1, 1, 0, 4, 4, 0, 1, 3, 4, 2, 0, 1, 4, 3,
      1, 2, 4, 2, 2, 2, 3, 3, 3, 0, 2, 0, 4, 1, 1, 3,
      0, 3, 1, 3, 4, 3, 3, 4, 0, 1, 0, 3, 3, 1, 4, 2,
      0, 3, 2, 3, 0, 4, 3, 1, 3, 3, 4, 3, 0, 0, 1, 0,
      2, 4, 1, 3, 1, 3, 2, 4, 2, 2, 0, 3, 2, 3, 1, 3,
      0, 2, 1, 3, 2, 2, 1, 3, 0, 2, 1, 3, 2, 2, 2, 0,
      0, 0, 3, 4, 1, 4, 4, 3, 3, 0, 1, 2, 4, 1, 4, 0,
      0, 4, 3, 2, 4, 3, 1, 2, 0, 4, 4, 2, 0, 4, 4, 4,
      4, 0, 1, 4, 4, 3, 0, 3, 2, 1, 4, 3, 0, 3, 0, 3,
      1, 3, 3, 2, 3, 2, 2, 2, 1, 0, 4, 2, 0, 4, 2, 2,
      0, 1, 0, 0, 2, 0, 3, 3, 2, 4, 0, 3, 1, 0, 3, 4,
      2, 4, 0, 1, 4, 1, 0, 0, 4, 3, 3, 1, 1, 4, 1, 3,
      1, 0, 4, 3, 3, 0, 2, 1, 3, 4, 1, 3, 3, 3, 0, 4,
      2, 3, 0, 0, 0, 1, 4, 3, 1, 4, 2, 0, 4, 2, 3, 0,
      1, 2, 0, 4, 0, 4, 4, 2, 1, 3, 4, 3, 2, 3, 3, 4,
      3, 2, 2, 1, 3, 0, 3, 2, 1, 0, 1, 3, 2, 0, 0, 0,
      1, 1, 2, 2, 0, 3, 1, 0, 3, 2, 0, 0, 2, 3, 0, 0,
      4, 4, 2, 0, 1, 1, 3, 0, 1, 0, 1, 1, 3, 4, 0, 0,
      3, 4, 4, 4, 0, 2, 4, 4, 1, 0, 2, 2, 3, 4, 4, 0,
      1, 3, 2, 4, 0, 1, 2, 1, 3, 3, 0, 3, 4, 1, 3, 1,
      0, 1, 0, 4, 4, 3, 4, 1, 0, 3, 1, 3]
```

This does look less lengthy but comes with a downside that it's hard to pin down a certain validator, say 123, from the list. With the original format, you can search the validator 123 with `ctrl+f`.

Another alternative is to break test cases in different YAML files. An `index.yaml` can include the summary of the test suite and valid tests, it can also leave the WIP tests away. 

```
01_shuffle/
├── index.yaml
├── test_case_1.yaml
├── test_case_2.yaml
└── test_case_3.yaml
```

**Comment by mratsim** [comment]
In the long-term, I think we should only keep minimal yaml in the repo and have the "complete" tests autogenerated as build artifacts.

This is to avoid having reviews like: https://github.com/ethereum/tests/pull/497/files (still OK) and https://github.com/ethereum/tests/pull/511/files (1 million lines).

For the second case an index.yaml shouldn't be necessary but if I had to read and debug manually I would prefer the first one.

**Comment by ChihChengLiang** [comment]
Fixed in #4
